app/src/main/res/values/strings.xml

```diff
@@ -114,7 +114,7 @@
     <string name="copyright_notes" translatable="false">
         Copyright 2013-2022 johnjohndoe.
         \nCopyright 2011-2015 Daniel Dorau.
-        \nContributions from 0x5ubt13, Adriano Pereira Junior, Akarsh Seggemu, Andrea Marziali, Andy Scherzinger, Andreas Schildbach, Animesh Verma, bashtian, bjoernb, Björn Olsson Jarl, ButterflyOfFire, cacarrara, Caio Volpato, cketti, codingcatgirl, entropynil, ideadapt, Jasper van der Graaf, Joergi, koelnkalkverbot, Larissa Yasin, ligi, Luis Azcuaga, Mateus Baptista, Matthias Geisler, Matthias Hunstock, MichaelRocks, Nghiem Xuan Hien, NiciDieNase, Noemis, Omicron, Poschi, rotrot, Sjors van Mierlo, Stefan Medack, SubOptimal, Teeranai.P, Thomaeum Kempen, Torsten Grote, Victor Herasme, Vladimir Alabov, Yanicka.
```

Comment:
Please also squash your changes into one commit and force push it onto this branch. You can simple `git reset HEAD~4` the commits on your branch and create a single new commit. Then `git push --force origin correctingContributionsNote` it to update this PR.

Reply:
I'll do that ASAP, the only problem is that my laptop's battery is currently dead 🤦